utils/scarb-ui/src/message.rs

```diff
@@ -16,7 +16,7 @@ pub trait Message {
     /// Return textual representation of this message.
     ///
     /// Default implementation returns empty string, making [`Ui`] skip printing this message.
-    fn text(self) -> String
+    fn text(&self) -> String
```

Comment:
this kinda contradicts the idea of messages being one-off. I know I forgot to document this principle, sorry for that